packages/material-ui-codemod/README.md

```diff
@@ -1067,6 +1068,67 @@ npx @material-ui/codemod@next v5.0.0/with-width <path>
 
 You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#material-ui-core-styles).
 
+#### `mui-replace`
+
+Replace every occurrence of `material-ui` related package with the the new package name (listed below) except these packages (`@material-ui/pickers`, `@material-ui/data-grid`, `@material-ui/x-grid` & `@material-ui/x-grid-data-generator`)
+
+**Material design components**
+
+```diff
+- import Alert from '@material-ui/core/Alert';
++ import Alert from '@mui/material/Alert';
```

Comment:
x the others

```suggestion
-import Alert from '@material-ui/core/Alert';
+import Alert from '@mui/material/Alert';
```

Reply:
I go with the first one and add this at the end.
`[More details about why package names are changed](https://github.com/mui-org/material-ui/issues/27666)`